integration-tests/ws-security-policy/src/main/resources/saml1-policy.xml

```diff
@@ -63,16 +41,18 @@
                     </sp:AlgorithmSuite>
                 </wsp:Policy>
             </sp:AsymmetricBinding>
+
             <sp:SignedElements>
                 <!-- This works, but setting io.quarkiverse.cxf.it.security.policy.SamlBeanProducers.SamlCallbackHandler.signAssertion = true does not -->
                 <sp:XPath xmlns:saml1="urn:oasis:names:tc:SAML:1.0:assertion">//saml1:Assertion</sp:XPath>
             </sp:SignedElements>
             <sp:SignedParts>
                 <sp:Body/>
             </sp:SignedParts>
+
             <sp:SupportingTokens xmlns:sp="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702">
                 <wsp:Policy>
-                    <sp:SamlToken sp:IncludeToken="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702/IncludeToken/AlwaysToRecipient">
+                    <sp:SamlToken sp:IncludeToken="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702/IncludeToken/Always">
                         <wsp:Policy>
```

Comment:
Similarly, as this policy xml will be shared between client and server, and it specifies to sign the SAML assertion element, so the SAML token should be available for both request and response message. 

I think here we may want separate policy xml for client and server to avoid this workaround.